client/src/dashboard/Dashboard.js

```diff
@@ -27,6 +27,7 @@ import RewritingRules from './RewritingRules';
 import RuleFormulator from './RuleFormulator';
 import QueryLogs from './QueryLogs';
 import JDBCDrivers from './JDBCDrivers';
+import ManageApplication from './ManageApplication';
```

Comment:
Since all other components use nouns as names, let's rename this component to `Applications`; (***But please verify if it has any conflicts with existing files.***)